packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -110,12 +110,13 @@ export class GCTelemetryTracker {
 			return false;
```

Comment:
Do you want to update the annotation above `private shouldLogNonActiveEvent` to reflect the change in logic?

Reply:
done.